aries-site/src/data/structures/components.js

```diff
@@ -1595,6 +1595,7 @@ export const components = [
   },
   {
     name: 'Tag',
+    accessibility: 'Failed WCAG 2.2 A',
```

Comment:
```suggestion
    accessibility: 'Passed WCAG 2.2 AA',
```

Now that the other rule is "passed" for the close icon, can this be changed?

Reply:
☝️ good catch!

Reply:
Yes! updated